plugins/woocommerce/includes/admin/views/html-admin-page-reports.php

```diff
@@ -35,7 +35,8 @@
 		?>
 	</nav>
 	<?php
-	if ( count( $reports[ $current_tab ]['reports'] ) > 1 ) {
+	$curr_tab_reports_count = is_countable( $reports[ $current_tab ]['reports'] ) ? count( $reports[ $current_tab ]['reports'] ) : 0;
+	if ( $curr_tab_reports_count > 1 ) {
```

Comment:
This one should be okay to revert.  The setup properly defines these as arrays.

Reply:
Done 318fe824600808a8a6bf6b97a406d86d4bb65d7c